accounts/views.py

```diff
@@ -1,3 +1,38 @@
-from django.shortcuts import render
+import json, re, bcrypt, jwt
 
-# Create your views here.
+from django.http import JsonResponse
+from django.views import View
+
+from accounts.models import Account
+
+from my_settings import SECRET_KEY
+
+
+class SignupView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+
+        EMAIL_REGES        = "^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$"
+        PASSWORD_REGES     = '[A-Za-z0-9@#$%^&+=]{8,}'
```

Comment:
```suggestion
        EMAIL_REGES    = "^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$"
        PASSWORD_REGES = '[A-Za-z0-9@#$%^&+=]{8,}'
```
= 기준으로 맞출 때 가장 긴 변수 기준으로 맞춰주면 됩니다.

Reply:
냅 수정했습니다